01week/datatypes.js

```diff
@@ -0,0 +1,69 @@
+//1. Write a JavaScript program to display the current day and time. getDay and toLocaleTimeString
+const dayTime = () => {
+  const d = new Date();
+  const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  const day = weekday[d.getDay()];
+  const time = d.toLocaleTimeString();
+
+  return day + ' ' + time;
+
+}
+console.log(dayTime());
+
+//2. Write a JavaScript program to convert a number to a string. toString method
+const numToString = (numb1) => {
+  const numString = numb1.toString();
+
+  return numString;
+}
+console.log(numToString(15));
+//3. Write a JavaScript program to convert a string to the number.  parseInt method
+const stringToNumber = (word) => {
+  const integer = parseInt(word, 10);
+
+  return integer
+}
+console.log(stringToNumber("43"));
+//4. Write a JavaScript program that takes in different datatypes and prints out whether they are a: Boolean, Null, Undefined, Number, NaN, String. typeof method
+
+const typeOfItem = (item) => {
+  return typeof item;
+}
+console.log(typeOfItem(45));
+//5. Write a JavaScript program that adds 2 numbers together.
+const addTwoNumbers = (num1, num2) => num1 + num2;
+
+console.log(addTwoNumbers(6, 3));
+//6. Write a JavaScript program that runs only when 2 things are true. Boolean function
+
+const bothAreTrue = (thing1, thing2) => {
+  if (Boolean(thing1) && Boolean(thing2)) {
```

Comment:
The Boolean method is redundant an unnecessary. Can be simplified to `if(thing1 && thing2)`